src/main/java/redis/clients/jedis/JedisCluster.java

```diff
@@ -83,14 +104,32 @@ public JedisCluster(HostAndPort node, final JedisClientConfig clientConfig, int
     this(Collections.singleton(node), clientConfig, maxAttempts, poolConfig);
   }
 
+  /**
+   * Creates a Jedis-Cluster with multiple Nodes/Instances.
```

Comment:
Here, one of the multiples nodes/instances is used as an entry point (or to make the first contact) with the cluster.